bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java

```diff
@@ -675,10 +675,12 @@ void removeCurCompactionLog() {
                 if (!compactionLogChannel.getLogFile().delete()) {
                     LOG.warn("Could not delete compaction log file {}", compactionLogChannel.getLogFile());
                 }
+
                 try {
-                    closeFileChannel(compactionLogChannel);
+                    compactionLogChannel.close();
                 } catch (IOException e) {
-                    LOG.error("Failed to close file channel for compaction log {}", compactionLogChannel.getLogId());
+                    LOG.error("Failed to close file channel for compaction log {}: {}", compactionLogChannel.getLogId(),
+                            e.getMessage());
```

Comment:
instead of logging just exception message, it would be better to log complete exception, right? we would get complete call stack trace aswell

LOG.error("Failed to close file channel for compaction log {}", compactionLogChannel.getLogId(), e);